### DISCUSSION TITLE
Transformation of points with transformix after correct registration

### DISCUSSION THREAD

**Discussion by IVarha** [SuperElastix/elastix#930] [Q&A]
Good day, I have mesh in moving space (I have mesh vertices generated from label in floating space).

Further, I do coregistration for images using elastix and get trasformparameters.txt. 

After I performed coregistration I visually checked it (generated image with _transformix_).

When I try to apply a transformation to points from mesh using _transformix_. My points transformed into wrong positions.

I tried to use simple code to do that same transformation. 

Here you can see what I mean, the transformed picture is not matched with the transformed label
![image](https://github.com/SuperElastix/elastix/assets/8555419/ea4e1eab-28c4-4e94-be27-3924d3f45c7e)

**Comment by ntatsisk** [comment]
Hi! You might find this discussion in the old mailing list helpful: https://groups.google.com/g/elastix-imageregistration/c/9YAv4iFHgZw/m/8APmPKY8AQAJ.

If your points are indeed defined in the moving image coordinate frame then you need to inverse your transform. You can find how to do that in section "6.1.6 DisplacementMagnitudePenalty: inverting transformations" of the elastix manual (download it from here: https://elastix.lumc.nl/doxygen/index.html)

Let us know how it goes and if the notebooks were helpful (we can update them if not).

Konstantinos
